programs/sbf/rust/invoke/src/processor.rs

```diff
@@ -1158,7 +1158,7 @@ fn process_instruction(
             let mut lamports = account.lamports();
             account
                 .lamports
-                .replace(unsafe { mem::transmute(&mut lamports) });
+                .replace(unsafe { mem::transmute::<&mut u64, &mut u64>(&mut lamports) });
```

Comment:
similar to https://github.com/anza-xyz/agave/pull/1343/files#r1601597230, I'd suggest:

```rust
// Forcibly extend lifetime of `lamports` to side-load into `accounts`.
account
    .lamports
    .replace(...transmute::<&'_ mut u64, &'static mut u64>(...)...)
```

Reply:
seems this diff is removed entirely. is it okay?

Reply:
sorry for missing this one. yeah... it is not okay 😢 just re-added

Reply:
oh, so, i think my previous review comment is still valid. i think `&mut u64, &mut u64` odd at the least.. @Lichtso: what do you think about https://github.com/anza-xyz/agave/pull/1343/files#r1601605299 ?

Reply:
just pushed a update: https://github.com/anza-xyz/agave/pull/1343/commits/e2a6a24ab89ad4cd501f82f82e9d9e3eddc9a96d

Reply:
heh, this is better fix. :)